certbot-apache/certbot_apache/_internal/override_arch.py

```diff
@@ -3,13 +3,14 @@
 
 from certbot import interfaces
 from certbot_apache._internal import configurator
+from certbot_apache._internal.configurator import _OsOptions
```

Comment:
Indeed, it does not make sense since we are already in a private package. For the record, and from what I observed, `pylint` is not complaining because it does not care about this naming convention when we are already inside a private package.